how-to/security/install-openvpn.md

```diff
@@ -120,12 +129,13 @@ ca ca.crt
 cert myservername.crt
 key myservername.key
 dh dh.pem
+tls-auth ./easy-rsa/ta.key 0
```

Comment:
Why the `./easy-rsa/` directory prefix? The `ta.key` file is being generated in `/etc/openvpn` by the instructions further down. I think this block should look just like the client config further down, with the `0` replaced with `1` of course.